src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -52,14 +52,11 @@ protected String getQuery(SlingHttpServletRequest request) {
         // is not returning any results
         StringBuilder queryBuilder = new StringBuilder()
                 .append("select m.* from [nt:base] as m ")
```

Comment:
You might want to try a `select distinct` instead if you want to remove duplicate entries

Reply:
Agreed, if that's an option, it seems ideal.